As a quick and easy work-around, if good genome annotation is not available for one of the genomes in question, one can use `NCBI Blast+` to find out which genes from genome A have a significant BLAST match in genome B. Note that this method is limited, as it will identify genes *likely* missing in the new genome, based on sequence but not on structure, and genes present in the new genome but not in the reference genome will not be found. 

For a local Blast of one genome against the other, the first step is to build a local database. `-in` is the input (genome, CDS or protein), `-dbtype` defines the type of database (nucl for nucleic acis, gene for CDS, protein for protein sequences), `-out` is the name of the database. 
```
makeblastdb -in PATH/StSwB9_denovo_.fa -dbtype nucl -out Sctrf_genome -title "Sc. trifolium genome"
```

To run the blast, I did the following:
```
/usr/local/bioinfo/src/NCBI_Blast+/ncbi-blast-2.6.0+/bin/blastn -evalue 0.001 -num_alignments 3 -query PATH/1980_CDS.fa -db PATH/to/db/Sctrf_genome -outfmt 7 -out blastn.out -num_threads 8 -negative_gilist Ss-genemodels_vs_St-genome.blastn.negative.out
```
Settings:
- `-evalue 0.001` sets the cut-off of the e-value to 0.001, so that only significant hits are returned. Can be more or less stringent. 
- `-num_alignments 3` limits the number of hits to the top3
- `-query` was all S. sclerotiorum CDSs here
- `-db` refers to the database defined above
- `-outfmt 7` means tabular output
- `-out` sets the prefix of the output files
- `negative_gilist` creates a file of queries without hit in the database


Then, I created a list of genes with hits in the database. This was done in three steps. First, `grep` to pull out all lines without `#`.
```
grep -v '#' blastn.out > blastn.tmp1.txt
```
To reduce the list to gene names (any other column can be added in `'{print $1}'`):
```
awk '!seen[$0]++' blastn.tmp1.txt | awk '{print $1}' > blastn.tmp2.txt
```
And finally, to remove duplicates:
```
awk '!x[$0]++' blastn.tmp2.txt > blastn.hits.list.txt
```
