This document is meant to summarize what input material I used and how I parsed the files to upload to an instance of `WebApollo`. 

## Input data
#### *Ab initio* gene predictions
`BRAKER` and `Augustus` produce gff annotation files of the predicted gene models. I have used the GeneMark and Augustus output produced by `BRAKER`, which needs parsing to gff3 as follows: 
```ShellSession
gffread -E MySu01_BRAKER-Augustus.gtf -o- > MySu01_BRAKER-Augustus.gff3
gffread -E MySu01_BRAKER-genemark.gtf -o- > MySu01_BRAKER-genemark.gff3
```
The gff3 file should look like this: 
```
# gffread -E genemark.gtf -o-
# gffread v0.10.1
##gff-version 3
tig00000001	GeneMark.hmm	mRNA	21	865	.	+	.	ID=1_t;geneID=1_g
tig00000001	GeneMark.hmm	exon	21	50	.	+	.	Parent=1_t
tig00000001	GeneMark.hmm	exon	147	865	.	+	.	Parent=1_t
tig00000001	GeneMark.hmm	CDS	21	50	.	+	2	Parent=1_t
tig00000001	GeneMark.hmm	CDS	147	865	.	+	2	Parent=1_t
```

#### Repeat landscape
text

#### BLAST similarities
text

#### RNA-seq coverage
text

#### RNA-seq reads
text

## Initializing genome in `WebApollo`
#### Setting up the genome
Text

## Uploading data
#### Annotations (Repeats, gene predictions, BLAST)
Text
